src/Compatibility/GenAPI/Microsoft.DotNet.GenAPI/GenAPIApp.cs

```diff
@@ -87,33 +80,5 @@ private static TextWriter GetTextWriter(string? outputDirPath, string assemblyNa
 
             return File.CreateText(outputDirPath);
         }
-
-        // Read the header file if specified, or use default one.
-        private static string ReadHeaderFile(string? headerFile)
```

Comment:
Moved to CSharpFileBuilder.cs as GenApiApp is the only class that uses CSharpFileBuilder. ApiDiff won't use this.